src/main/java/tds/exam/results/mappers/ExamineeMapper.java

```diff
@@ -31,14 +31,26 @@
         "SSID", "StudentIdentifier"
     );
 
+    private static final Map<String, String> relationshipNames = ImmutableMap.of(
```

Comment:
Would it make sense to consolidate this map with the `attributesMap`?  I realize that conceptually attributes and relationships are two different things, but both of these maps accomplish the same goal; translate one name to another.  Maybe name it `attributesAndRelationshipsTranslationMap` or something?